srcpkgs/rust/template

```diff
@@ -19,7 +19,7 @@ maintainer="Enno Boland <gottox@voidlinux.eu>"
 homepage="https://www.rust-lang.org/"
 license="MIT, Apache-2.0"
 distfiles="https://static.rust-lang.org/dist/rustc-${version}-src.tar.gz"
-checksum=1d5a81729c6f23a0a23b584dd249e35abe9c6f7569cee967cc42b1758ecd6486
+checksum=5088e796aa2e47478cdf41e7243fc5443fafab0a7c70a11423e57c80c04167c9
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" cargo llvm"
```

Comment:
how about adding `rust>=${version}_${revision}` here? This will make sure rust isn't installed in an older version.

Reply:
Wouldn't this trigger builds for the host arch on all builders when updating rust?

Reply:
Also, Manual.md says:
`There is no need to specify a version because the current version in srcpkgs will always be required.`
So I'm actually sort of confused why it doesn't pull in the newest rust anyway

Reply:
@Cogitri true, the version can (and must) be skipped. But the build is only triggered if it is explicitly added.

@jnbr That's true. Looking at it logically rust-cross needs the same version to succeed. I hope that dxpb handles this case more intelligent.